src/doc/src/reference/unstable.md

```diff
@@ -1230,66 +1230,134 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
 
 ### workspace-inheritance
 
-* RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
-* Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
 
-The `workspace-inheritance` feature allows workspace members to inherit fields
-and dependencies from a workspace.
+### The `workspace.package` table
 
-Example 1:
+This would be in [`workspaces.md`][workspaces], under
```

Comment:
Please prefix this with `Stablization:` and italicize it so its clear this is for the unstable-docs reader and not for what will be in the docs.

While the content makes it clear, being able to scan for information is important